common/views/components/SearchTabs/SearchTabs.tsx

```diff
@@ -2,18 +2,28 @@ import BaseTabs, { TabType } from '../BaseTabs/BaseTabs';
 import { classNames, font } from '@weco/common/utils/classnames';
 import styled from 'styled-components';
 import Space from '../styled/Space';
-import { useContext, useState, FunctionComponent, ReactElement } from 'react';
+import {
+  useContext,
+  useState,
+  FunctionComponent,
+  ReactElement,
+  useEffect,
+} from 'react';
+import NextLink from 'next/link';
 import { AppContext } from '../AppContext/AppContext';
 import PrototypeSearchForm from '@weco/common/views/components/PrototypeSearchForm/PrototypeSearchForm';
 import {
   WorksRouteProps,
   ImagesRouteProps,
+  imagesLink,
 } from '@weco/common/services/catalogue/ts_routes';
 import {
   CatalogueAggregationBucket,
   CatalogueAggregations,
 } from '@weco/common/model/catalogue';
 import { trackEvent } from '@weco/common/utils/ga';
```

Comment:
trackEvent probably not be relevant now as the items are links ? Would this still work as the ga is dealing with the page tracking.